### PR TITLE
Paypal SDK Integration

### PR DESCRIPTION
**Pull request by Peter0892**
Uses the PayPal SDK PHP to integrate the library with the application.
https://github.com/paypal/PayPal-PHP-SDK/wiki

### CODE REVIEW

File: server.iml [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ghiobi) [inline server.iml]: Please .gitignore this file and remove this file on Github.

File: app/Providers/PayPalServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     *
+     * @return void
+     */
+    public function register()
+    {
+        // Add paypal library as singleton
+        $this->app->singleton(PayPalServiceProvider::class, function($app){
+            return new \PayPal\Rest\ApiContext(
+                new \PayPal\Auth\OAuthTokenCredential(
+                    'ATPZcJYf7Ob4M9pH4CSFbpVhqwxW5HH1rntthIiSETmQMQWhDgDDS-1UBP1_PDBlHNjDKaV_8nLyw-3b',     // ClientID
```
Comment (ghiobi) [inline app/Providers/PayPalServiceProvider.php]: Please used `config('services.paypal.client_id')` and `config('services.paypal.client_secret')`. 
You are exposing API keys on Github.

**Comment by Peter0892** [comment]
Made requested modifications

**Comment by ghiobi** [comment]
@JTalarico Please merge.

**Pull request merged**